test/test.oauth2.ts

```diff
@@ -736,9 +736,19 @@ function mockExample() {
 
 it('should refresh token if missing access token', (done) => {
   const scopes = mockExample();
-  client.credentials = {refresh_token: 'refresh-token-placeholder'};
+  let raisedEvent = false;
+  const refreshToken = 'refresh-token-placeholder';
+  client.credentials = {refresh_token: refreshToken};
+
+  // ensure the tokens event is raised
+  client.on('tokens', tokens => {
+    assert.equal(tokens.access_token, 'abc123');
```

Comment:
would be great to have `abc123` in a variable and use it both in mock in the top of the file and in all asserts.